content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -357,7 +395,7 @@ The isolation levels configured by default in each database are:
 | CockroachDB | `Serializable`   |
 | SQLite      | `Serializable`   |
 
-For more database-specific information on isolation levels, see the following resources:
+For database-specific information on isolation levels, see the following resources:
 
 - [Transaction isolation levels in PostgreSQL](https://www.postgresql.org/docs/9.3/runtime-config-client.html#GUC-DEFAULT-TRANSACTION-ISOLATION)
 - [Transaction isolation levels in Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-transaction-isolation-level-transact-sql?view=sql-server-ver15)
```

Comment:
I think I asked before when this was created: Why do we only list PostgreSQL and SQL Server and not the other databases? Did we turn that into a follow up issue?

Reply:
I don't think we have an issue, but I'll create one.